simple_certmanager/models.py

```diff
@@ -60,6 +70,21 @@ class Meta:
             )
         ]
 
+    def create_certificate(self, certificate):
```

Comment:
following the django code style I linked earlier, custom methods go _below_ the `__init__` and `save` (and other built in methods)